test/integration/queue/producers/QueueProducer-test.js

```diff
@@ -14,7 +14,7 @@ const messageParser = require('~/queue/util/message-parser')
 const Event = require('~/models/events/Event')
 const waitForEvent = require('~/test/util/waitForEvent')
 
-const AMQ_URL = 'amqp://localhost'
+const AMQ_URL = 'amqp://rabbitmq'
```

Comment:
Hmm, it works with either `amqp:rabbitmq` or `amqp://rabbitmq`? Surprising, but a future pr that consolidates it would be nice.